scripts/python.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# This script correctly installs Python dependencies
+set -eu
+
+# XXX: Ideally, we should make this import relative rather than from where the Makefile
+# invokes this
+# shellcheck disable=SC1091
+source scripts/lib.sh
+
+ensure-venv() {
+    ./scripts/ensure-venv.sh
+}
+
+upgrade-pip() {
+    # pip versions before 20.1 do not have `pip cache` as a command which is necessary for the CI
+    pip install --no-cache-dir --upgrade "pip>=20.1"
+}
+
+install-py-dev() {
+    ensure-venv
+    upgrade-pip
+    # The Python version installed via pyenv does not come with wheel pre-installed
+    # Installing wheel will speed up installation of Python dependencies
+    require wheel || pip install wheel
+    echo "--> Installing Sentry (for development)"
+    pip_version=$(pip -V |  awk '{print $2}')
+    # Older versions of pip require SYSTEM_VERSION_COMPAT in Big Sur
+    # shellcheck disable=SC2072
+    if query_big_sur && [[ $pip_version < 20.3 ]]; then
```

Comment:
It's not correct to do this, this is a lexicographical comparison so I can `[[ 3.1 < 20.1 ]] || echo fail` because the `.` in the `3.1` is greater than `0`. Similarly, `[[ 21.0.9 < 21.0.10 ]] || echo fail`.

Reply:
Thanks for bringing this up!
I will look into it.